bin/dfx-software-sns-aggregator-version

```diff
@@ -20,17 +20,27 @@ source "$SOURCE_DIR/clap.bash"
 # Define options
 clap.define short=l long=pinned desc="Get the pinned version from versions.bash (default)" variable=GET_PINNED nargs=0
 clap.define short=l long=latest desc="Get the latest version" variable=GET_LATEST nargs=0
```

Comment:
Note: There is a bug above where the short form `-l` is repeated.  This error has been replicated across several commands so is fixed in a separate PR.